### PR TITLE
Release/sample app circle

### PR DESCRIPTION
**Pull request by dmkr**

### CODE REVIEW

**Review by mnorth719 — APPROVED** [review]
LGTM!

**Review by douglasjsellers — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      
+workflows:
+  version: 2
+  beta_build:
+    jobs:
+      - build:
+          filters:
+            branches:
+              only:
+                - /Release\/.*/
```
Comment (theBradfo) [inline .circleci/config.yml:43]: We should really grab this file from `burner-android`, the workflow is so much cleaner!

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Gemfile
+source "https://rubygems.org"
+
+gem 'fastlane', '2.102.0'
```
Comment (theBradfo) [inline Gemfile:4]: Also fastlane stinks

File: app/src/main/java/adhoclabs/co/burnersamplesdk/SampleActivity.java [hunk 3]
```diff
@@ -29,6 +31,7 @@ protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_sample);
     burnerSDK = BurnerSDK.getInstance();
+    Fabric.with(this, new Crashlytics());
```
Comment (theBradfo) [inline app/src/main/java/adhoclabs/co/burnersamplesdk/SampleActivity.java:34]: This should go in the `Application` subclass, so you're not constrained to this activities context.

**Pull request closed without merging**